actors/verifreg/src/lib.rs

```diff
@@ -142,11 +144,16 @@ impl Actor {
         let verifier_addr = Address::new_id(verifier);
 
         rt.transaction(|st: &mut State, rt| {
+            let existing_cap =
+                st.get_verifier_cap(rt.store(), &verifier_addr)?.ok_or_else(|| {
+                    actor_error!(illegal_argument, "verifier {} not found", verifier_addr)
+                })?;
```

Comment:
I don't think adding a new state read is an acceptable cost to get this field in. I'm skeptical we should add "previous" values, but in this case if we decide that we do, please thread it out of the remove_verifier call instead (it's returned from the Map delete operation).

Reply:
@anorth Have consensus with @rvagg that we don't need to add previous values anymore. We've changed the FIP update proposal accordingly.